lib/node/runner/checker_test.go

```diff
@@ -61,20 +61,31 @@ func TestOnlyValidTransactionInTransactionPool(t *testing.T) {
 		targetAccount, targetKP := TestMakeBlockAccount(common.Amount(10000000000000) /* 100,00000 BOS */)
 		targetAccount.MustSave(nodeRunner.Storage())
 
-		tx := transaction.TestMakeTransactionWithKeypair(networkID, 1, rootKP, targetKP)
+		tx := transaction.TestMakeTransactionWithKeypair(networkID, 1, rootKP, false, targetKP)
 		tx.B.SequenceID = rootAccount.SequenceID
 		tx.Sign(rootKP, networkID)
 
 		runChecker(tx, nil)
-
 		require.True(t, nodeRunner.TransactionPool.Has(tx.GetHash()), "valid transaction must be in `Pool`")
 	}
 
+	{ // invalid transaction: invalid fee
```

Comment:
What purpose of this testcase?
there is no assertion here.

Reply:
I missed to remove this. Now test for fee is in checker_message_test.go